.github/actions/pack_and_deploy/action.yml

```diff
@@ -7,48 +7,21 @@ runs:
         # Our testing expects that the init process (PID 1) will
         # reap orphan processes. At least the following test leans
         # on it: app-tap/gh-4983-tnt-e-assert-false-hangs.test.lua.
-        echo PACKPACK_EXTRA_DOCKER_RUN_PARAMS='--init' | tee -a $GITHUB_ENV
-        # There are packages like tzdata or postfix, whose configuration
-        # is interactive by default. The environment variable below
-        # fordibs interactive configuration phase. Otherwise the CI stucks
-        # from time to time on `apt-get update`.
-        export DEBIAN_FRONTEND=noninteractive
-        # create packages
+        export PACKPACK_EXTRA_DOCKER_RUN_PARAMS='--init'
```

Comment:
Whether ` | tee -a $GITHUB_ENV` is removed intentionally? If so, please, clarify.

Reply:
Yes, it was intentional. The `| tee -a $GITHUB_ENV` construction is used to propagate an env variable outside the action so it could be used in other actions and workflow steps.

Reply:
Hm. Seems so. Either move it into its own commit or at least comment why do you change it in the commit message.

Reply:
Ok, will do.

Reply:
Fixed